dev/tasks/r/github.packages.yml

```diff
@@ -324,6 +329,7 @@ jobs:
         env:
           LIBARROW_BINARY: "FALSE"
           ARROW_R_DEV: "TRUE"
+          CMAKE_FIND_DEBUG_MODE: "ON"

```

Comment:
Should we remove this before merge?

Reply:
I want to keep this to debug a problem in the future.
This step is executed only when the above build step is failed. Is it accepted?

Reply:
:+1: makes sense!